doc/Requirements.md

```diff
@@ -0,0 +1,526 @@
+High level requirements
+=======================
+
+(Note: some requirements are only motivated by the fact that the reference implementation
+       implements them. Seach for "reference implementation".)
+
+# Introduction
+The PF MUST be a library.
+(Why: To be reused in different components.)
```

Comment:
I do not understand your wish. Is what is currently generated by pandoc fits you ?